gating/check/run_deploy_mnaio.sh

```diff
@@ -128,7 +128,12 @@ source /opt/rpc-openstack/RE_ENV
 # RO-4206
 # Use fork of Ansible which exposes the apt errors so that we
 # can diagnose the cause of the apt fetch failures.
-export ANSIBLE_PACKAGE="git+https://github.com/rcbops/ansible@v2.4.4.0-with_apt_errors"
+# export ANSIBLE_PACKAGE="git+https://github.com/rcbops/ansible@v2.4.4.0-with_apt_errors"
```

Comment:
If we are comment this out why not just drop it?

Reply:
Gating might need it, according to dave. I deleted it in my upstream patch in deploy.sh...